.travis.yml

```diff
@@ -17,85 +12,83 @@ before_install:
 install:
   -
 
-env:
-  -JDK=Java8
-  -JDK=Java9
-  -JDK=Java10
-  -JDK=Java11-EA
-  -JDK=Java12-EA
-
-matrix:
+jobs:
   fast_finish: true
   include:
     - jdk: oraclejdk8
       env:
         - DESC="acceptance tests"
-        - JDK=Java8
         - CMD="mvn install --projects acceptance-tests --also-make --activate-profiles all"
 
     - jdk: oraclejdk8
       env:
         - DESC="findbugs"
-        - JDK=Java8
         - CMD="mvn install findbugs:check --projects '!scala-unit-tests,!jmh-scala-tests,!jmh-tests,!jcstress-tests,!p2-repository' --activate-profiles all -DskipTests=true"
+
     - jdk: oraclejdk8
       env:
         - DESC="checkstyle"
-        - JDK=Java8
         - CMD="mvn install checkstyle:check --activate-profiles all -DskipTests=true"
 
     - jdk: oraclejdk8
       env:
         - DESC="unit tests"
-        - JDK=Java8
         - CMD="mvn install"
 
     - jdk: oraclejdk8
       env:
         - DESC="compile performance-tests"
-        - JDK=Java8
         - CMD="mvn install --projects performance-tests --also-make --activate-profiles all -DskipTests=true"
 
     - jdk: oraclejdk9
       env:
-        - DESC="unit tests Java 9"
-        - JDK=Java9
+        - DESC="unit tests oraclejdk9"
         - CMD="mvn install"
 
-    - jdk: oraclejdk8
+    - jdk: openjdk10
```

Comment:
does declaring it as openjdk10 work? I do not know, asking for information.

Reply:
The travis docs are a little scattered. This works:

https://docs.travis-ci.com/user/languages/java/#using-java-10-and-later

I interpret the docs to mean that these jdks are installed lazily on demand. Not that we have to invoke `install-jdk.sh` because we definitely don't.